Lib/test/test_re.py

```diff
@@ -2303,6 +2303,8 @@ def test_possessive_quantifiers(self):
         self.assertIsNone(re.match("^x{}+$", "xxx"))
         self.assertTrue(re.match("^x{}+$", "x{}"))
 
+        self.assertEqual(re.match('((.(?!C))++)', 'ABCD').span(), (0, 1))
```

Comment:
You may add a comment `# gh-100061` for this test-case.

Reply:
Added :)